CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Add unofficial plugin `react-static-plugin-file-watch-reload` to plugins list
 - Enable configuring css loader from `react-static-plugin-sass` and `react-static-plugin-less` ([#1348](https://github.com/react-static/react-static/pull/1348))
 - Update `react-static-plugin-jss` for react-jss v10+. ([#1367](https://github.com/react-static/react-static/pull/1367))
+- Add inline script hashes to `DocumentProps`. These hashes can be used to construct a Content Security Policy in a meta tag without `unsafe-inline` scripts.
```

Comment:
Add the PR link please :)

Reply:
Oops, done in https://github.com/react-static/react-static/pull/1373/commits/fa8d0c00f56e006405df494188fb308e9042d986